src/make_data/banner_intervention/banner_selects_results_page.sql

```diff
@@ -0,0 +1,44 @@
+/* This query extracts the total number and proportion of sessions that access the
+   checker via the banner intervention that access the results page
+*/
+
+
+DECLARE start_date STRING DEFAULT "20210803";
+DECLARE end_date STRING DEFAULT "20210816";
+
+CREATE OR REPLACE TABLE `govuk-bigquery-analytics.banner_intervention.banner_results_page` AS
+
+-- All distinct sessions that select the checker via the banner (eventAction =
+-- interventionClicked)
+
+WITH
+  sessions_select_banner AS (
+    SELECT DISTINCT
+      CONCAT(fullVisitorId, "-", visitId) AS sessionId
+    FROM `govuk-bigquery-analytics.87773428.ga_sessions_*`
+    CROSS JOIN UNNEST(hits) AS hits
+    WHERE _TABLE_SUFFIX BETWEEN start_date AND end_date
```

Comment:
```suggestion
    WHERE _TABLE_SUFFIX BETWEEN start_date AND end_date
    AND hits.eventInfo.eventCategory = 'interventionBanner'
```

Reply:
Updated to include: in commit https://github.com/alphagov/govuk-entity-personalisation/pull/36/commits/77d71f59b52253592b35330d57a6a25d755e3e40

Reply:
Many thanks - have added as within the other scripts in commit https://github.com/alphagov/govuk-entity-personalisation/pull/36/commits/77d71f59b52253592b35330d57a6a25d755e3e40